eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php

```diff
@@ -25,12 +25,13 @@ public static function dumpAssets(CommandEvent $event)
         $options = self::getOptions($event);
         $appDir = $options['symfony-app-dir'];
         $webDir = $options['symfony-web-dir'];
-        $env = isset($options['ezpublish-asset-dump-env']) ? $options['ezpublish-asset-dump-env'] : '';
+        // @deprecated: Use of ezpublish-asset-dump-env is deprecated, use standard SYMFONY_ENV instead.
+        $env = !empty($options['ezpublish-asset-dump-env']) ? $options['ezpublish-asset-dump-env'] : (getenv('SYMFONY_ENV') ?: '');
 
         if (!$env) {
             $env = $event->getIO()->ask(
-                "<question>Which environment would you like to dump production assets for?</question> (Default: 'prod', type 'none' to skip) ",
-                'prod'
+                "<question>'SYMFONY_ENV' environment variable not defined, which environment would you like to dump production assets for?</question> (Default: 'dev', type 'none' to skip) ",
+                'dev'
```

Comment:
This sounds both major and minor... I'm not sure I understand that part. Why does it matter of `update` or `install` script is executed ? Did this affect the environment we run things for ?

What bothers me is that this `prod` / `dev` change _is_ a behaviour change (I'm thinking automatic scripts).